src/components/DatasetPageHeader.js

```diff
@@ -3,10 +3,10 @@ import { Box } from 'grommet'
 import { usePollDatasetStatus } from 'hooks/usePollDatasetStatus'
 import { useResponsive } from 'hooks/useResponsive'
 import getDatasetState from 'helpers/getDatasetState'
+import { DatasetNotProcessed } from 'components/DatasetNotProcessed'
+import { DatasetProcessingError } from 'components/DatasetProcessingError'
+import { DatasetRegenerate } from 'components/DatasetRegenerate'
```

Comment:
If these are all headers shouldn't we rename them to `DatasetNotProcessedHeader` and then this component just renders the correct one for the state. I think this should also apply to `DatasetProcessing` and `DatasetReady`